src/main/java/org/jenkinsci/plugins/pipeline/utility/steps/json/WriteJSONStepExecution.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2016 CloudBees Inc.
```

Comment:
I didn't comment on the copyright of the other step's files since they where about 90% copy & paste so I deemed that to be correct.